pytorch_lightning/core/lightning.py

```diff
@@ -479,6 +479,7 @@ def log_dict(
         sync_dist_op: Optional[Any] = None,  # noqa: Remove in 1.6
         sync_dist_group: Optional[Any] = None,
         add_dataloader_idx: bool = True,
+        rank_zero_only: Optional[bool] = None,
```

Comment:
could you add the docs for this as well? pretty much the same text as in the log() above